_posts/2023-12-07-semantic-options-benchmarks.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: post
+title:  "Semantic Search with OpenSearch: Architecture options and Benchmarks"
+authors:
+ - sean-zheng-amazon
+ - zane-neo
```

Comment:
'zane-neo' author profile does not exist - who should this be? Please confirm/fix